### ISSUE TITLE
What is listed as v2 examples doesn't match the new v2

### ISSUE THREAD

**Issue by deanm0000** [yokawasa/azure-functions-python-samples#56]
It seems microsoft called one thing v2 over a year ago so that now the new v2 programming model is refering to a different versioning scheme.

IE. https://learn.microsoft.com/en-us/azure/azure-functions/functions-reference-python?tabs=wsgi%2Capplication-level&pivots=python-mode-decorators****

**Comment by rcarmo** [comment]
Yep. This is a bit confusing. I too am looking for samples for the v2 programming model.
